service/openapi.yaml

```diff
@@ -329,6 +329,19 @@ components:
           properties:
             cacheInvalidation:
               $ref: '#/components/schemas/featureCacheInvalidation'
+            evaluation:
+              $ref: '#/components/schemas/featureEvaluation'
+    featureEvaluation:
+      type: object
+      description: Configuration for the cache cacheInvalidation
```

Comment:
this seems like a copy paste of the description below, is this correct?

Reply:
oh good catch, let me change that